centreon/packages/js-config/cypress/e2e/commands.ts

```diff
@@ -413,6 +448,30 @@ interface ListingRequestResult {
   };
 }
 
+interface PatchBody {
```

Comment:
PatchBody is too generic
```suggestion
interface PatchDashboardBody {
```